app/javascript/containers/searchUserListContainer.js

```diff
@@ -41,10 +41,10 @@ function mapStateToProps(state) {
     error: searchUserList.error,
     officesList: officesList(state),
     inputData: searchUserList.inputData,
-    firstName: searchUserList.inputData.firstName,
-    email: searchUserList.inputData.email,
-    CWSLogin: searchUserList.inputData.CWSLogin,
-    lastName: searchUserList.inputData.lastName,
+    firstName: searchUserList.inputData.firstName ? searchUserList.inputData.firstName : '',
```

Comment:
doesn't safeGet do this for us?  import safeGet from 'lodash.get' then...
```suggestion
   firstName: safeGet(searchUserList, 'inputData.firstName', '')
```

Reply:
yeah! good call out, isn't better to use the safeGet in the selector instead here in container?

Reply:
Sure.  That sounds even better.